legalize-vadd.mir

```diff
@@ -3,16 +3,56 @@
 # RUN: | FileCheck %s
 
 ---
-name:		test_vadd
+name:		test_nxv1s8
 body:		 |
   bb.0.entry:
-    ; CHECK-LABEL: name: test_vadd
-    ; CHECK: [[COPY:%[0-9]+]]:_(<vscale x 1 x s32>) = COPY $v8
-    ; CHECK: [[COPY1:%[0-9]+]]:_(<vscale x 1 x s32>) = COPY $v9
-    ; CHECK-NEXT: [[VADD:%[0-9]+]]:_(<vscale 1 x s32>) = G_ADD [[COPY]], [[COPY1]]
-    ; CHECK-NEXT: PseudoRET implicit VADD
+    %0:_(<vscale x 1 x s8>) = COPY $v8
```

Comment:
Is this case missing test checks?

Reply:
I was also wondering, seems like all `v1` test cases don't have automatic test checks

Reply:
They should have test checks. Can you investigate what is needed so that test checks are generated for all test cases in the test file?

Reply:
Sure!

Reply:
> They should have test checks. Can you investigate what is needed so that test checks are generated for all test cases in the test file?

It was funky, basically it's because I used tabs `\t` insteaed space ` ` for the `v1` test cases, and the `update_mir_test_checks.py` state transition just go unexpectedly, resulting in no checks generated ...